src/requirements.txt

```diff
@@ -1,40 +1,61 @@
 alembic==0.9.5
+amqp==2.3.2
 aniso8601==1.3.0
-atomicwrites==1.2.1
-attrs==18.2.0
-autopep8==1.4
+appnope==0.1.0
+astroid==2.0.4
+Babel==2.6.0
+backcall==0.1.0
+billiard==3.5.0.3
 blinker==1.4
+celery==4.2.1
 certifi==2018.1.18
 chardet==3.0.4
 click==6.7
 coverage==4.5
 coveralls==1.2.0
+decorator==4.3.0
 docopt==0.6.2
 ecdsa==0.13
-Flask==0.12.2
+flake8==3.6.0
+Flask==0.12.4
```

Comment:
Hey, @mutoromike  I wanted to suggest that for dependencies that were there before, we stick to their former version unless we want to intentionally change it after confirming that the version we are changing to is stable for our application.

Reply:
Thanks for the concern, this was changed due to Flask-CLI's requirements.

Reply:
In that case, then we can keep it that way 👍